zhaquirks/tuya/__init__.py

```diff
@@ -1128,6 +1129,15 @@ class TuyaZBExternalSwitchTypeCluster(CustomCluster):
     attributes = {0xD030: ("external_switch_type", ExternalSwitchType)}
 
 
+# Tuya Zigbee Cluster 0x1888 Implementation
+class TuyaZB1888Cluster(CustomCluster):
+    """Tuya manufacturer specific cluster 6280."""
+
+    name = "Tuya Manufacturer Specific 1"
+    cluster_id = TUYA_CLUSTER_1888_ID
+    ep_attribute = "tuya_manufacturer_specific_1"
```

Comment:
Not happy with that name, but don't have any nice suggestion. Maybe `ep_attribute = "tuya_manufacturer_specific_1888"`? or `"tuya_manufacturer_specific_6280"`?

Reply:
Changed it to append the decimal representation of the cluster ID now: https://github.com/zigpy/zha-device-handlers/pull/2592/commits/a061fa6a7537c564e894ecf374eafd8c27c8e6c4
.. or would hex be better?

Reply:
I don't recall any similar 'labeling'.
Any approach would be fine to me, maybe the hex would direct match to the device signature, isn't it?

Reply:
I'll leave it in decimal for now. For hex, it might be confusing if we don't have `0x` in front(?)
Doesn't really matter and we can always change it later, should we ever need to use the `ep_attribute` of those two clusters.